website/client/src/components/groups/group.vue

```diff
@@ -225,12 +226,15 @@
     box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
     padding: 1em;
     text-align: center;
-    min-width: 80px;
-    max-width: 120px;
+    min-width: 120px;
     height: 76px;
 
+    &:last-of-type {
+      margin-right: 1rem;
+    }
+
     .svg-icon.shield, .svg-icon.gem {
-      width: 28px;
+      min-width: 28px;
```

Comment:
I think this should be set back to `width` not `min-width`, it's causing some issues with firefox
![image](https://user-images.githubusercontent.com/589911/95336087-86935180-08b0-11eb-934b-61f5db078699.png)

Reply:
Yeah, that shouldn't have been there, thanks for the catch. Must have done that by accident.